cornflow-dags/DAG/rostering/core/instance.py

```diff
@@ -662,14 +683,15 @@ def get_incompatible_slots_employee(self) -> TupList:
             ("2021-09-06T21:00", "2021-09-07T07:00", 1), ...]
         """
         availability = self.get_employees_ts_availability()
-        return TupList(
+        ts_tuple = TupList(
             [
                 (ts, ts2, e)
                 for (ts, ts2) in self._get_incompatible_slots()
                 for e in self._get_employees("name")
                 if (ts, e) in availability and (ts2, e) in availability
             ]
         )
+        return ts_tuple
```

Comment:
Why this change?

Reply:
I like it better and helps me when debugging since i have an object to review